kdwsdl2cpp/common/fileprovider.cpp

```diff
@@ -41,7 +41,7 @@ static QHash<QUrl, QByteArray> fileProviderCache;
 
 
 FileProvider::FileProvider()
-  : QObject( 0 )
+  : QObject( nullptr )
```

Comment:
This constructor can be `= default` or removed altogether.

Reply:
I know, but I didn't wanted to change the basic layout, otherwise there are also some destructors which could be replaced.